viewing_party/party.py

```diff
@@ -1,23 +1,143 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:

```

Comment:
Excellent use of the guard clause pattern here! You could make this a bit more "Pythonic" using `not`, e.g.
```python
if not title or not genre or not rating:
```